pages/profile/edit_email.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { NativeBaseProvider } from "native-base";
+import {StyleSheet, Text, TextInput, View, SafeAreaView, TouchableOpacity} from 'react-native';
+import { NativeStackNavigationProp } from "@react-navigation/native-stack";
+import { AntDesign } from '@expo/vector-icons'; 
+
+
+type RootStackParamList = {
+    Edit: undefined;
+    EditEmail: undefined;
+};
+
+type editEmailScreenNavigationProp = NativeStackNavigationProp<
+    RootStackParamList,
+    "EditEmail"
+>;
+
+type Props = {
+    navigation: editEmailScreenNavigationProp;
+};
+
+const EditEmail = ({ navigation }: Props) => {
+
+    const [email, setEmail]= React.useState("");
+    
+    return (
+    <NativeBaseProvider>
+        <SafeAreaView style={styles.container}>
+            <View style={styles.header}>
+                <TouchableOpacity
+                    onPress={()=>navigation.navigate('Edit')}>
+                    <AntDesign name="leftcircle" size={40} color='#88838A' />
```

Comment:
I think you shld replace this with Material Community icons.

We're gonna be using Material Community icons in our app cos they have more icons and variations and our overall UI/UX design is also gonna be leaning towards the Material design philosophies so we'll go with that for consistency. 

If the icon you're trying to find doesn't exist in material community than you can also try finding it in material icons.